data/helpers.d/package

```diff
@@ -124,6 +124,9 @@ ynh_install_app_dependencies () {
     	manifest_path="../settings/manifest.json"	# Into the restore script, the manifest is not at the same place
     fi
     version=$(grep '\"version\": ' "$manifest_path" | cut -d '"' -f 4)	# Retrieve the version number in the manifest file.
+    if [ " ${#version}" -eq 0 ]; then
+        version="1.0"
+    if
```

Comment:
I think it would look much nicer with a `fi` here 😉

Reply:
Are you sure ?
I think bash would like the idea of changing its habits.